charts/core/src/Chart/Chart.styles.ts

```diff
@@ -12,10 +19,67 @@ export const chartContainerStyles = css`
   width: 100%;
 `;
 
-export const chartHeaderContainerStyles = css`
-  grid-area: chartHeader;
+const getDraggableContainerStyles = (
+  transform: Transform | null,
+  transition?: string,
+) => css`
+  transform: ${CSS.Transform.toString(transform)};
```

Comment:
I assume this is because of how dndKit returns a transform value?  Can we just add a comment about this?